understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
     <section id="brief">
 			<h2>In brief</h2>
 			<dl>
-				<dt>Goal</dt><dd>Content can be enlarged without increasing line length.</dd>
-				<dt>What to do</dt><dd>Make lines of text reflow within the viewport.</dd>
-        <dt>Why it's important</dt><dd>People who need bigger text find it difficult if they must scroll to read long lines.</dd>
+				<dt>Goal</dt><dd>Content can be enlarged without needing to scroll in two dimensions.</dd>
```

Comment:
the reason this was changed was [per feedback](https://docs.google.com/document/d/1PV5IB5p7ps9pUhvQOZ7KyG4yooRbVDTNYkTJpCdiYVw/edit?disco=AAABQsG2ei4) from @alastc 
>I probably should have raised this before, but line-length wasn't the motivation per-se.
>It was more that you don't lose information from the view in two directions.
>E.g. Content can be enlarged without needing to scroll in two directions.

when i updated this to say "in two directions", i received feedback from two other people asking "why are you changing the wording from what the SC says"

maybe we should take another stab at redoing this line, since there's conflicting recommendations now.